c2corg_ui/static/js/documentediting.js

```diff
@@ -0,0 +1,269 @@
+goog.provide('app.DocumentEditingController');
+goog.provide('app.documentEditingDirective');
+
+goog.require('app');
+goog.require('ol.format.GeoJSON');
+goog.require('ol.geom.Point');
+
+
+
+/**
+ * @constructor
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.documentEditingDirective = function() {
+  return {
+    restrict: 'A',
+    scope: true,
+    controller: 'appDocumentEditingController',
+    controllerAs: 'editCtrl',
+    bindToController: true
+  };
+};
+
+
+app.module.directive('appDocumentEditing', app.documentEditingDirective);
+
+
+
+/**
+ * @param {angular.Scope} $scope Scope.
+ * @param {angular.JQLite} $element Element.
+ * @param {angular.Attributes} $attrs Attributes.
+ * @param {angular.$http} $http
+ * @param {string} apiUrl Base URL of the API.
+ * @constructor
+ * @ngInject
+ * @export
+ */
+app.DocumentEditingController = function($scope, $element, $attrs, $http,
```

Comment:
The editing controller is used to edit a document but also to create a new document? Maybe you can add a short note in the comment block (e.g. "Controller to create a new document and to edit an existing document."). Just to make it clear.